ouroboros-consensus/src/Ouroboros/Consensus/Util/Orphans.hs

```diff
@@ -38,6 +37,9 @@ import           Ouroboros.Network.AnchoredFragment (AnchoredFragment)
 import qualified Ouroboros.Network.AnchoredFragment as AF
 import           Ouroboros.Network.MockChain.Chain (Chain (..))
 
+import           Cardano.Binary
```

Comment:
This is another place where the new imports are not grouped according to https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md

(Thank you for your patience 😬.)